### PR TITLE
OSDOCS-6304: Documented the 4.11.42 zstream release notes

### PR DESCRIPTION
**Pull request by dfitzmau**
[OSDOCS-6304](https://issues.redhat.com/browse/OSDOCS-6304)

Version(s):
4.11

Link to docs preview:
[Preview link](https://60647--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-11-release-notes.html#ocp-4-11-42)

### CODE REVIEW

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -3581,3 +3581,22 @@ $ oc adm release info 4.11.41 --pullspecs
...
+
+[source,terminal]
+----
+$ oc adm release info 4.11.42 --pullspecs
+----
+
+[id="ocp-4-11-42-upgrading"]
+==== Updating
+
+To update an existing {product-title} 4.11 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster within a minor version by using the CLI] for instructions.
```
Comment (agantony) [inline release_notes/ocp-4-11-release-notes.adoc:3602]:
```suggestion
To update an existing {product-title} 4.11 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
```

**Comment by agantony** [comment]
Hi @dfitzmau Overall LGTM, I only added a small suggestion, thx!

**Comment by skrthomas** [comment]
merge review lgtm

**Pull request merged**